phoenicis-dist/src/scripts/phoenicis-create-package.sh

```diff
@@ -62,7 +62,45 @@ fi
 
 if [ "$PHOENICIS_OPERATING_SYSTEM" == "Linux" ]; then
     jpackager create-image "${PHOENICIS_JPACKAGER_ARGUMENTS[@]}"  --linux-bundle-name "phoenicis-playonlinux"
-    jpackager create-installer deb "${PHOENICIS_JPACKAGER_ARGUMENTS[@]}" --linux-package-deps "unzip, wget, xterm | x-terminal-emulator, imagemagick, cabextract, icoutils, p7zip-full, curl, winbind" --linux-deb-maintainer "PlayOnLinux Packaging <packages@playonlinux.com>" --linux-bundle-name "phoenicis-playonlinux"
+
+    packageName="Phoenicis_$VERSION"
+    cd "$PHOENICIS_TARGET"
+    rmdir packages/PhoenicisPlayOnLinux/
+    rm -rf "packages/phoenicis" 2> /dev/null
+    mv packages/Phoenicis\ PlayOnLinux/ packages/phoenicis
+    rm -rf "$packageName" 2> /dev/null
+    mkdir -p "$packageName/DEBIAN/"
+
+    cat << EOF > "$packageName/DEBIAN/control"
```

Comment:
Should we maybe move this to its own file?

Reply:
Not easily, the version is dynamic

Reply:
(And also later the installed size)